cg/meta/invoice.py

```diff
@@ -14,15 +14,35 @@ def __init__(self, db: Store, lims_api: lims.LimsAPI, invoice_obj: models.Invoic
         self.log = []
         self.invoice_obj = invoice_obj
         self.customer_obj = invoice_obj.customer
+        self.record_type = ''
+        self.raw_records = []
+        self._get_record_type()
+
+
+    def _get_record_type(self):
```

Comment:
you're not getting anything with this method, e.g. there is no return statement. You are however setting the record_type. Why not name it that way?

Reply:
There is no more record_type 'Prov'?

Reply:
oh, I see. You renamed that?